cumulusci.yml

```diff
@@ -125,6 +143,9 @@ plans:
             - when: "'.my.' not in org_config.instance_url"
               action: error
               message: "Please enable My Domain in your org prior to installing."
+            - when: "not tasks.check_enable_networks_enabled()"
+              action: warn
+              message: "In order to include the Outbound Funds Module Site Template in your installation, please ensure that Digital Experiences is available in your org prior to installing."
```

Comment:
@nicolersmith Can you review this installer message? This warning should appear in the installer if you don't have Digital Experiences enabled in your org. Without that setting enabled, the OFM Site Template installation would fail.

![Screen Shot 2021-03-16 at 11 49 33 AM](https://user-images.githubusercontent.com/1134606/111356173-b7d93000-864d-11eb-8f37-7ef0b6b1e03d.png)

Reply:
Hey @seanpat09, just noticed this today. Can you update this installer message to:

"After enabling Digital Experiences, create, configure, and then activate your new Experience Builder site so it's visible to your users."